(assert-exception EvaluationException (let (('x 2)) x))